tests/admin_docs/test_views.py

```diff
@@ -293,6 +294,25 @@ def test_model_docstring_renders_correctly(self):
     def test_model_detail_title(self):
         self.assertContains(self.response, '<h1>admin_docs.Person</h1>', html=True)
 
+    def test_app_not_found(self):
+        response = self.client.get(
```

Comment:
Single line looks okay here, in the next test and in the assertEqual of the test.